Makefile

```diff
@@ -181,6 +195,10 @@ lint-charts: $(HELM) # Run helm lint tests
 	helm lint --strict charts/secrets-store-csi-driver
 	helm lint --strict manifest_staging/charts/secrets-store-csi-driver
 
+.PHONY: shellcheck
+shellcheck: $(SHELLCHECK)
+	find . -name '*.sh' -not -path './docker/*' | xargs $(SHELLCHECK)
```

Comment:
Excluding `docker` dir as that'll trigger a image build and we don't want to bump to rc for this change. I'll fix the errors in `docker` dir in a later PR and then we would be able to make this `$(SHELLCHECK) */*.sh`

Reply:
https://github.com/kubernetes-sigs/secrets-store-csi-driver/issues/558 to track this cleanup